log4j-core/src/test/java/org/apache/logging/log4j/core/Log4j1222Test.java

```diff
@@ -34,8 +34,13 @@
 	@Test
 	public void homepageRendersSuccessfully()
 	{
-        System.setProperty("log4j.configurationFile", "log4j2-console.xml");
-		Runtime.getRuntime().addShutdownHook(new ShutdownHook());
+		System.setProperty("log4j.configurationFile", "log4j2-console.xml");
+		try {
+			// TODO: Does not work correctly; when shutdown hook fails test has already finished successfully
```

Comment:
Added this comment because test might be misleading. Test "failures" would most likely not actually cause test to fail.